cli/src/semgrep/app/scans.py

```diff
@@ -391,3 +392,6 @@ def report_findings(
             raise Exception(
                 f"API server at {state.env.semgrep_url} returned this error: {response.text}"
             )
+
+        ret = response.json()
+        return (ret.get("exit_code", 0), ret.get("reason", ""))
```

Comment:
does the app ever not send `exit_code`?

Reply:
it currently does not 😓 and better to be backwards compatible if we decide to stop